### DISCUSSION TITLE
Allow folks to enter brand colors without #

### DISCUSSION THREAD

**Discussion by michael-tharrington** [forem/forem#16121] [Feature Requests]
This is just a tiny thing, but you get different behavior based on where you are...

If you try to enter a brand color without `#` for your user profile (available at the bottom of the page on `/settings`) and click to save, then your profile reverts to default colors.

If you try to enter a brand color without `#` for an org (`/settings/organization`) and click to save, you get the error "Text color hex is invalid".

I think ideally we would allow folks to enter these colors with or without `#` and the color would be accepted either way. Perhaps it'd be autocorrected to include the `#`.

**Comment by citizen428** [comment]
IMO we shouldn't require users to enter colors as strings but use HTML color selects instead.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/color

**Comment by nickytonline** [comment]
We are in the process of updating inputs for colors. It's a combo of a textbox and color input (the control you can click on to select a colour). We have both, because currently the native colour input is not accessible in all browsers. cc: @aitchiss
